index.html

```diff
@@ -219,6 +219,119 @@ <h3>Skills</h3>
 				</div>
 			</div>
 		</section>
+		<footer id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							placeholder="Full name"
+							required
+						/>
+					</label>
+
+					<label
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label>
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
+				</fieldset>
+			</form>
+			<form
+				action="https://formspree.io/f/xpznwejk"
+				method="post"
+				class="form2"
+			>
+				<fieldset>
+					<label class="firstN"
+						><input
+							type="text"
+							name="firstN"
+							id="firstN"
+							placeholder="First name"
+							required
+						/>
+					</label>
+					<label class="lastN">
+						<input
+							type="text"
+							name="lastN"
+							id="lastN"
+							placeholder="Last Name"
+							required
+						/>
+					</label>
+
+					<label class="email"
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label class="msg">
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
+				</fieldset>
+			</form>
+			
+		</footer>
+
+	<footer id="copyright">
+		<div class="copyright">
+				<ul class="socials">
+					<li>
+						<a href=""><i class="fa-brands fa-github"></i></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+					</li>
+					<li>
+						<a href=""><img src="/assets/Vector.png" alt="vector" /></a>
```

Comment:
- Please fix the directory for the `vector.png` icon. I suggest you use `src="./assets/Vector.png"`. Please fix this 🙏